tools/integration_tests/util/operations/file_operations.go

```diff
@@ -192,3 +194,38 @@ func ReadFileSequentially(filePath string, chunkSize int64) (content []byte, err
 	}
 	return
 }
+
+func WriteFileSequentially(filePath string, fileSize int64, chunkSize int64) (err error) {
+	var numberOfBytes int
+	var offset int64 = 0
+
+	file, err := os.OpenFile(filePath, os.O_RDWR|syscall.O_DIRECT|os.O_CREATE, FilePermission_0600)
+	if err != nil {
+		log.Printf("Error in opening file:%v", err)
+	}
+
+	// Closing file at the end.
+	defer CloseFile(file)
+
+	for offset < fileSize {
+		// Get random 200MB or remaining filesize data into chunk.
```

Comment:
200 MB?

Reply:
chunk size is 200MB so we are first getting chunk size data or the data remaining to write in file(<200MB)

Reply:
Okay, we can remove 200MB from the comment. As it's a generic method, only knows about chunkSize.